RELEASES.md

```diff
@@ -143,6 +143,7 @@ Security releases are published on a 3-monthly cycle, around the mid point betwe
 | 2024 Q2 | 2024/06/04 | 2024/06/04 |   0 days   |
 | 2024 Q3 | 2024/09/03 | 2024/09/19 |  16 days   |
 | 2024 Q4 | 2024/12/03 | 2024/12/18 |  15 days   |
-| 2025 Q1 | 2025/03/04 |            |            |
+| 2025 Q1 | 2025/03/04 | 2025/03/20 |  16 days   |
+| 2025 Q2 | 2025/03/03 |            |            |
```

Comment:
06/03?

Reply:
argh - yep - thats why we need code review 8/

Reply:
fixed